lib/Core/Executor.h

```diff
@@ -439,6 +439,7 @@ class Executor : public Interpreter {
                              const char *suffix = NULL,
                              const llvm::Twine &longMessage = "");
   
+  void emitPmemError(ExecutionState &state, const std::unordered_set<std::string> &errors);
```

Comment:
My only concern is that you may re-emit bugs multiple times with this, since an un-flushed cache line may stay unflushed in later states. What are you doing to mitigate that?